engine/admin-api/adapter/gql/resolver.go

```diff
@@ -210,6 +210,17 @@ func (r *queryResolver) Versions(ctx context.Context, productID string) ([]*enti
 	return r.versionInteractor.ListVersionsByProduct(ctx, loggedUser, productID)
 }
 
+func (r *queryResolver) ProcessRegistries(ctx context.Context, productID string, processType *string) ([]*entity.ProcessRegistry, error) {
```

Comment:
We are retrieving a list of processes (ore registered processes), not process registries.
I'd rename the method to reflect that.

```suggestion
func (r *queryResolver) RegisteredProcesses(ctx context.Context, productID string, processType *string) ([]*entity.ProcessRegistry, error) {
```